Automattic/wp-calypso
# Style Variation: Use global-styles endpoint to apply the selected variation

#### Proposed Changes

* This PR is focusing on switching to the `global-styles` endpoints to apply the variation.
* Note that the `global-styles` endpoints only support the active theme for now, so we have to do it after we activated the theme.

#### Testing Instructions



* Head to the logged-in Theme Showcase `/themes/${your_site}`. If using calypso.live, the flag `themes/showcase-i4/details-and-preview` is required.
* Find a theme with style variation, open the theme preview modal, and select one of the styles in the sidebar.
* Activate the theme and ensure that your site has the theme with the selected style applied.

Note that the selected style variation should be applied successfully on both simple site and atomic site 🙂

#### Pre-merge Checklist





Related to https://github.com/Automattic/wp-calypso/pull/71754
